src/facepalm/core.clj

```diff
@@ -347,13 +347,17 @@
       "1.2.0:20120101.01"
       ver)))
 
+(defn- sortable-version-keyfn [version-str]
+  (vec (map #(Integer/parseInt %) (string/split version-str #"[.:]"))))
```

Comment:
Oh, that's a good idea. I'd originally just written `map`, got an error because it was a lazy sequence and it can't use that as a comparator key, then added `vec` without thinking if I should combine things. Thanks!